catalogue/webapp/components/SearchFilters/SearchFilters.Desktop.DynamicFilters.tsx

```diff
@@ -37,7 +37,18 @@ const DynamicFilterArray = ({
   };
 
   const renderDynamicFilter = (f: Filter, i: number, arr: Filter[]) => {
-    return (
+    // We need to have the excluded filters still in the form so their values gets retained in the URL
+    // when more filtering is done (e.g. partOf.title)
+    // Only checkbox types are excluded at the moment
```

Comment:
(the other types are date range and colour which are very specific to one filter each.)